PreSignUpLambdaHook/certs/eu-west-2-bundle.pem

```diff
@@ -0,0 +1,124 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I assume this file comes from AWS somehow and allows us to be confident we're talking to the write DB.
How does the DB know it's allowed to talk to us? Permissions on the lambda?

Reply:
Permissions, and DB credentials passed to the Lambda via environment variables (the package `pg` automatically uses env vars like `PGDATABASE` and `PGPASSWORD` which is why the code doesn't show them being imported)